WebRoot/public/login.jsp

```diff
@@ -778,6 +781,19 @@ function onLoad() {
         loginForm.totpcode.focus();
 	}
 }
+
+ function reloadImage() {
+     var xhttp = new XMLHttpRequest();
+     var captcha_id;
+     xhttp.onreadystatechange = function() {
+     if (this.readyState == 4 && this.status == 200 && this.responseText != null && this.responseText != "") {
+        this.captcha_id = this.responseText;
+        document.images['imageName'].src = '${varCaptchaApiUrl}/captcha/' + this.captcha_id + '.png';
```

Comment:
related to the above comment. If the name 'imageName' is changed, 

    document.images['imageName'].src

also needs to be changed.